W-18829709-api-alert-note-duke (#399)
mulesoft/docs-monitoring: modules/ROOT/pages/basic-alerts.adoc

* *Resource name* (apps), *Server name* (servers), *API* (APIs)
+
Application, server, or API to apply the alert to
+

> For APIs, you must also provide an API version. Only active versions of an API are selectable. Inactive APIs do not submit metrics. 

Reviewer: ```suggestion
For APIs, you must also provide an API version. Only active versions of an API are available for alerts. 
```

Author: They see the versions, and they may need more context.
